src/libutil/util.hh

```diff
@@ -737,4 +754,11 @@ inline std::string operator + (std::string && s, std::string_view s2)
     return std::move(s);
 }
 
+inline std::string operator + (std::string_view s1, const char * s2)
+{
+    std::string s(s1);
+    s.append(s2);
```

Comment:
Probably not very important, but in case it is, I guess we could avoid a copy with something like

```suggestion
    std::string s;
    s.reserve(s1.size() + strlen(s2));
    s.append(s1);
    s.append(s2);
```